.gitignore

```diff
@@ -335,4 +335,6 @@ qwiki.db
 
 #appsettings.json
 appsettings\.json
-appsettings\.Development\.json
\ No newline at end of file
+appsettings\.Development\.json
+WikidataGame/WikidataGame.Firebase.TokenProvider/q-wiki-57251382-firebase-adminsdk-8whmj-64e446ddda.json
```

Comment:
How do I get a token now if I want to run the server locally?

Reply:
You are now required to use the TokenProvider project I added, which creates you a token from a userid. This needs some configuration though. Not really an awesome solution, but Firebase makes our authentication a hell lot more complex.

Reply:
Might be good to add a word on that to the README?

Reply:
That's true, but the AuthSecret has been necessary for quite some time now, this is nothing I changed in this PR. I will put it in the readme anyway.